starr/modules/help.py

```diff
@@ -65,8 +65,9 @@ async def _default_help(ctx: tanjun.abc.MessageContext) -> None:
             message_commands.append(
                 (
                     "|".join(command.names),
-                    command.metadata["help"] + "\n" +
-                    f"Subcommands: {', '.join('|'.join(c.names) for c in command.commands)}",
+                    command.metadata["help"]
+                    + "\n"
+                    + f"Subcommands: {', '.join('|'.join(c.names) for c in command.commands)}",
```

Comment:
I know I wrote this lol, but lets combine the newline into the f string.

```suggestion
                    + f"\nSubcommands: {', '.join('|'.join(c.names) for c in command.commands)}",
```

Reply:
same